### PR TITLE
Adds facet for admin set; closes #1605.

### LINKED ISSUE #1605: Add facet for admin set

### PR DESCRIPTION
**Pull request by coblej**
Also adds related helper methods and updates .gitignore to not
commit config/locales/admin_set.yml .

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,6 +23,9 @@
 # Environment configs
 /config/environments/*.rb
 
+# Ignore certain translation files
```
Comment (dchandekstark) [inline .gitignore]: Why is this added? We're not using it anymore ...

**Comment by dchandekstark** [comment]
OK, I see. I think we missed some communication.  Admin sets are now managed in ddr-aux, so the "full name" for an admin set is admin_set.title.  See https://github.com/duke-libraries/ddr-models/blob/develop/lib/ddr/models/admin_set.rb.

**Comment by coblej** [comment]
Good catch.  I'll see about fixing this tomorrow.  Probably also means we should change or deprecate this: https://github.com/duke-libraries/ddr-models/blob/45ce55c1a24e75c4612d13702480e919d457d0aa/app/helpers/models_helper.rb#L3-L5
